bin/anthology/latexcodec.py

```diff
@@ -1,3 +1,19 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Marcel Bollmann <marcel@bollmann.me>
```

Comment:
This file's not mine. It was originally written by David Eppstein, and the main source appears to be [this code recipe on ActiveState](http://code.activestate.com/recipes/252124-latex-codec/), where it claims to be PSFL licensed. Furthermore I think our version of this file has been heavily modified by @mjpost, @danielgildea, and @davidweichiang (but most notably -- not me :)).

Reply:
Probably we can just swap that license in the header here, then?

https://docs.python.org/3/license.html

Reply:
I'll update the pull request on Monda or Tuesday.